### PR TITLE
Tek830 a 16 add interface elements for displaying lamps turning on and off

### PR DESCRIPTION
**Pull request by RezaAmanii**
Added follwing images:
- Bulb on & off
- Switch on & off

Added the Lamp class which represent a Lamp and switch

Added different method to be able to open the images and locate them in the main frame

Added a click handler method which make the bulb ON and OFF when clicked on it

### CODE REVIEW

**Review by erbergli — CHANGES_REQUESTED** [review]
When the file paths are written like this "TEK830/Images/xxxxx", I get this error: FileNotFoundError: [Errno 2] No such file or directory: 'TEK830/Images/OFF.png'.
Maybe use relative path instead?

**Comment by RezaAmanii** [comment]
> When the file paths are written like this "TEK830/Images/xxxxx", I get this error: FileNotFoundError: [Errno 2] No such file or directory: 'TEK830/Images/OFF.png'. Maybe use relative path instead?

I have used relative path for each image

**Comment by erbergli** [comment]
Works now! But someone else should probably also review incase I missed something

File: View/__pycache__/AppFrame.cpython-312.pyc
Comment (filipkauffeldt) [inline View/__pycache__/AppFrame.cpython-312.pyc:1]: This should not be included. Should be in gitignore

File: View/__pycache__/Lamp.cpython-312.pyc
Comment (filipkauffeldt) [inline View/__pycache__/Lamp.cpython-312.pyc:1]: This should not be included. Should be in gitignore

File: View/Lamp.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import tkinter as tk
+from PIL import Image, ImageTk
+import os
+
```
Comment (filipkauffeldt) [inline View/Lamp.py]: Remove new lines

File: View/Lamp.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+class Lamp:
+    def __init__(self, parent):
+
+        # Current state
+        self.currentState = "OFF"
+
+        # Image directory (cross-platforms path)
+        image_dir = os.path.join("Images")
+
```
Comment (filipkauffeldt) [inline View/Lamp.py]: Remove new line

File: Images/OFF.png
Comment (filipkauffeldt) [inline Images/OFF.png:1]: What is the source of these images? Do we have the right to use them?

File: View/__pycache__/AppFrame.cpython-312.pyc
Comment (RezaAmanii) [reply View/__pycache__/AppFrame.cpython-312.pyc:1]: Now it is in gitignore

File: View/__pycache__/Lamp.cpython-312.pyc
Comment (RezaAmanii) [reply View/__pycache__/Lamp.cpython-312.pyc:1]: Now it is in gitignore

File: Images/OFF.png
Comment (RezaAmanii) [reply Images/OFF.png:1]: Downloaded them from freepik.com, it is free to use

File: View/Lamp.py (hunk 2 above)
Comment (RezaAmanii) [reply View/Lamp.py]: Removed

File: View/Lamp.py (hunk 1 above)
Comment (RezaAmanii) [reply View/Lamp.py]: Removed

**Pull request closed without merging**